### PR TITLE
MUNI tech writers: timeout documentation.

### PR DESCRIPTION
**Pull request by sourn00dl**
Documentation edit regarding the issue [#881](https://github.com/ansible/ansible-documentation/issues/881).
Followup from ansible documentation PR [#1282](https://github.com/ansible/ansible-documentation/pull/1282).

### CODE REVIEW

File: lib/ansible/keyword_desc.yml [hunk 1]
```diff
@@ -61,7 +61,7 @@ serial: Explicitly define how Ansible batches the execution of the current play
 strategy: Allows you to choose the strategy plugin to use for the play. See :ref:`strategy_plugins`.
 tags: Tags applied to the task or included tasks, this allows selecting subsets of tasks from the command line.
 tasks: Main list of tasks to execute in the play, they run after :term:`roles` and before :term:`post_tasks`.
-timeout: Time limit for the task to execute in, if exceeded Ansible will interrupt and fail the task.
+timeout: Time limit for the task to execute in, if exceeded, Ansible will interrupt the process. Timeout does not affect templating or looping.
```
Comment (bcoca) [inline lib/ansible/keyword_desc.yml]: `for the task action to execute in` as 'the task' means a lot larger scope to most, I overlooked that when i first wrote the docs.

File: lib/ansible/config/base.yml [hunk 2]
```diff
@@ -1921,7 +1921,11 @@ TASK_TIMEOUT:
   name: Task Timeout
   default: 0
   description:
-    - Set the maximum time (in seconds) that a task can run for.
+    - Set the maximum time (in seconds) for a task to execute in.
```
Comment (bcoca) [inline lib/ansible/config/base.yml]: `for a task action to execute in`

File: lib/ansible/config/base.yml [hunk 3]
```diff
@@ -1921,7 +1921,11 @@ TASK_TIMEOUT:
...
+    - Timeout runs independently from templating or looping. 
+      It applies per each attempt and remains unchanged by the total time spent on a task.
```
Comment (bcoca) [inline lib/ansible/config/base.yml]: It applies to each attempt of executing the task's action and ...

File: lib/ansible/config/base.yml [hunk 4]
```diff
@@ -1921,7 +1921,11 @@ TASK_TIMEOUT:
...
+    - When the task execution exceeds the timeout, Ansible interrupts the process.
```
Comment (bcoca) [inline lib/ansible/config/base.yml]: When the action execution

**Comment by sourn00dl** [comment]
@oraNod I've encountered an issue with the latest push, some checks didn't pass. I checked the log, but still can't figure it out, could you please take a look and help me troubleshoot what might be causing these failures?

**Comment by oraNod** [comment]
@sourn00dl Hey, I think the failures are down to some transient issues with the test environment and not something to do with the changes in this PR. You can see the failures in lines like this one: https://dev.azure.com/ansible/ansible/_build/results?buildId=110896&view=logs&j=39f5381f-b636-5081-f62f-619f7b163664&t=70f85b7b-3a3b-5c97-e68d-42eb46ed4e7d&l=32875

I'd say there's nothing you can do other than sit tight and wait for someone to kick off a new run. I don't have permissions to any of the ci workflows in this repo and can't really do much either.

**Comment by sourn00dl** [comment]
@oraNod Alright, thank you! In the meantime, when it  comes to the changes to the documentation, is there something that would need some additional reworking?

**Comment by webknjaz** [comment]
The RHEL failures are unrelated and at to be fixed separately.

**Comment by sourn00dl** [comment]
@bcoca Hello, may I please ask for a review of the latest changes?

File: lib/ansible/keyword_desc.yml [hunk 5]
```diff
@@ -61,7 +61,7 @@ serial: Explicitly define how Ansible batches the execution of the current play
 strategy: Allows you to choose the strategy plugin to use for the play. See :ref:`strategy_plugins`.
 tags: Tags applied to the task or included tasks, this allows selecting subsets of tasks from the command line.
 tasks: Main list of tasks to execute in the play, they run after :term:`roles` and before :term:`post_tasks`.
-timeout: Time limit for the task to execute in, if exceeded Ansible will interrupt and fail the task.
+timeout: Time limit for the task action to execute in, if exceeded, Ansible will interrupt the process. Timeout does not affect templating or looping.
```
Comment (bcoca) [inline lib/ansible/keyword_desc.yml]: 'affect' seems to be the wrong word here, timeout does not account for/ include/cover  templating and looping

**Review by bcoca — APPROVED** [review]

**Review by samccann — APPROVED** [review]

**Review by s-hertel — APPROVED** [review]

**Comment by s-hertel** [comment]
@sourn00dl Thank you!

**Pull request merged**